inc/sailag.h

```diff
@@ -141,6 +141,14 @@ typedef enum _sai_lag_attr_t
      */
     SAI_LAG_ATTR_DROP_TAGGED,
 
+    /**
+     * @brief LAG Name
+     *
+     * @type char
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
```

Comment:
just an info, that this breaks backward compatibility, since users will have to add this flag for every lag, maybe it would be better to mark this as "create_only" ? to preserver backward compatibility

Reply:
@kcudnik Should we be adding a name for the 'port' object as well for consistency ?

Reply:
ports can be uniquely identified by lanes

Reply:
@kcudnik  If the flag is "create_only", it would need to assign default value. This attribute should be unique and no suitable default value for this attribute.

Reply:
this attribute is also mandatory_on_create so it should not have default value